docs/publishers/core-datasets.md

```diff
@@ -0,0 +1,135 @@
+# Core Datasets
+
+**Core Datasets** - important, commonly-used datasets in high quality, easy-to-use & open form.
+
+**High Quality & Reliable** - sourcing, normalizing and quality checking a set of **key reference and indicator datasets such as country codes, currencies, GDP and population**
```

Comment:
We've never discussed it, but should we use standard (British) English or American English? For instance, here we have "normalizing", which is spelt in American.